fbw-a32nx/src/systems/extras-host/index.tsx

```diff
@@ -40,18 +43,6 @@ class ExtrasHost extends BaseInstrument {
      */
     private gameState = 0;
 
-    constructor() {
-        super();
-
-        this.bus = new EventBus();
-        this.hEventPublisher = new HEventPublisher(this.bus);
-
-        this.versionCheck = new VersionCheck(this.bus);
-        this.keyInterceptor = new KeyInterceptor(this.bus);
-
-        console.log('A32NX_EXTRASHOST: Created');
-    }
```

Comment:
Is there a specific reason you prefer to avoid the construcor?

I'd at least want to keep the log output so finding issues is easier.